sensor_msgs/Image roi_image
float64[] roi_values

